docs/en/status-communities/understand-token-requirements-in-communities.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 490
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Understand token requirements in communities
+
+<!--
+image
+-->
+
+In Status, there are two kinds of communities: public and private. If you wish to join a private community, you need to meet the community owner token requirements. To maintain access to a private community, hold the necessary tokens at all times.
```

Comment:
'wish' -> 'want'?
'the community owner token requirements' -> 'the token requirements set by the community owner'